README.md

```diff
@@ -351,9 +351,11 @@ See an [example config here](#example-config). Provide the config when you call
 
 **appPageURL**: The default URL to navigate to when `withAuthUser` or `withAuthUserTokenSSR` need to redirect to the app. Optional unless using the `AuthAction.REDIRECT_TO_APP` auth action.
 
-**loginAPIEndpoint**: The API endpoint to call when the auth state changes for an authenticated Firebase user. **Required.**
+**loginAPIEndpoint**: The API endpoint to call when the auth state changes for an authenticated Firebase user. Must be set unless `tokenChangedHandler` is set.
 
-**logoutAPIEndpoint**: The API endpoint to call when the auth state changes for an unauthenticated Firebase user. **Required.**
+**logoutAPIEndpoint**: The API endpoint to call when the auth state changes for an unauthenticated Firebase user. Must be set unless `tokenChangedHandler` is set.
+
+**tokenChangedHandler**: A callback that runs when the auth state changes for a particular user. Cannot be set with `loginAPIEndpoint` or `logoutAPIEndpoint`.
```

Comment:
Let's specify more info here:
* Why this might be useful: "Use this if you want to customize how your client-side app calls your login/logout API endpoints (for example, to use a custom fetcher or add custom headers)."
* Add what the user is responsible for in this case: 1) calling their login/logout endpoints depending on the user's auth state, 2) passing the user's ID token in the `Authorization` header, 3) ensuring it allows the request to set cookies.
* Specify it's called when the user's ID token changes, equivalent to Firebase's [onIdTokenChanged](https://firebase.google.com/docs/reference/js/firebase.auth.Auth#onidtokenchanged) event